var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -11,14 +11,15 @@ class PyScikitLearn(PythonPackage):
     """A set of python modules for machine learning and data mining."""
 
     homepage = "https://scikit-learn.org/"
-    pypi = "scikit-learn/scikit-learn-0.24.0.tar.gz"
+    pypi = "scikit-learn/scikit_learn-1.5.0.tar.gz"
     git = "https://github.com/scikit-learn/scikit-learn.git"
 
-    maintainers("adamjstewart")
-
     license("BSD-3-Clause")
+    maintainers("adamjstewart")
```

Comment:
@rgommers I realize we're starting to get farther and farther from your wheelhouse, but do you have any interest in maintaining sklearn or pandas? Both have ported to meson following a similar setup as numpy/scipy/matplotlib.

Reply:
Sure, why not:) Both are a little more in my wheelhouse than Matplotlib actually, and I am an occasional contributor to both.

Reply:
Sklearn and pandas have way fewer bug reports than numpy/scipy/matplotlib, so I think they should be easy to maintain.